framework/raw-rsa-keyring.md

```diff
@@ -57,10 +61,16 @@ On keyring initialization, the following inputs are REQUIRED:
 
 A UTF-8 encoded value that namespaces this keyring.
 
+The raw RSA keyring MUST NOT accept a key namespace of "aws-kms".
+
+This value is also used for bookkeeping purposes.
```

Comment:
This is probably fine (since it looks like it was also part of the Raw AES keyring). But always good to clarify this further. Does "bookkeeping" mean "auditing" or something else in this context?

Reply:
I think we should look at #135 for how we use these.
I think that bookkeeping is too weak :)
I think that they exists _so that_ the keyring can recognize the encrypted data keys on decrypt.
That is not just bookkeeping (I think?)

Reply:
Hum thinking more, I'm inclined to keep this so the current spec is consistent.
But I think that #135 or a followup should come back and address this.